Code/GraphMol/RGroupDecomposition/testRGroupDecomp.cpp

```diff
@@ -2182,6 +2182,29 @@ void testUnlabelledRGroupsOnAromaticNitrogen() {
   }
 }
 
+void testNoTempLabels() {
+  BOOST_LOG(rdInfoLog)
+      << "********************************************************\n";
+  BOOST_LOG(rdInfoLog) << "Test that temp labels are removed from results"
+                       << std::endl;
+  auto core = "[1*]c1ccccc1"_smiles;
+  auto mol = "Cc1ccccc1"_smiles;
+  RGroupDecompositionParameters params;
+  RGroupDecomposition decomp(*core, params);
+  TEST_ASSERT(decomp.add(*mol) == 0);
+  TEST_ASSERT(decomp.process());
+  auto rows = decomp.getRGroupsAsRows();
+  TEST_ASSERT(rows.size() == 1);
+  const auto &res = rows.front();
+  for (const auto &pair : res) {
+    for (const auto a : pair.second->atoms()) {
+      for (const auto &propName : a->getPropList()) {
+        TEST_ASSERT(propName.find("label") == std::string::npos);
+      }
```

Comment:
This isn't wrong, but why not just directly test for the two properties you cleared?

Reply:
Because it's actually 4 properties:
* `RLABEL_TYPE`
* `RLABEL`
* `RLABEL_CORE_INDEX`
* `SIDECHAIN_RLABELS`
And, most importantly, because only `RLABEL` is visible and exported. So I thought that rather than hardcoding the literals of the labels (which might at some point change and break the test) it would be simpler to test for lowercase `label` which is present in all those temp labels while all permanent labels have sentence-case `Label`.